components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -225,12 +415,22 @@ fn migrate_logins(
     let tx = conn.unchecked_transaction()?;
 
     for login in migration_plan.logins.values() {
-        // Migrate local login first
+        // Could not easily use an equality here due to the inner value
+        // But neccesary to ensure we log proper metrics
+        match &login.migration_op {
```

Comment:
This should be `if let` (I expect clippy will complain about that too, and will probably tell you exactly what to replace it with too :) I suspect that's what you were getting at in the comment, but I think it will work fine (and if so, the comment should also be removed)